Packs/OctoxLabs/Integrations/OctoxLabs/OctoxLabs.yml

```diff
@@ -0,0 +1,217 @@
+          description: 'Discovery progress'
+          type: Number
+    - name: get-last-discovery
+      deprecated: false
+      description: 'Get last success discovery'
+      execution: false
+      outputs:
+        - contextPath: OctoxLabs.Discovery.id
+          description: 'Last discovery id'
+          type: Number
+        - contextPath: OctoxLabs.Discovery.start_time
+          description: 'Last discovery start time'
+          type: String
+        - contextPath: OctoxLabs.Discovery.end_time
+          description: 'Last discovery end time'
+          type: String
+        - contextPath: OctoxLabs.Discovery.status
+          description: 'Last discovery status'
+          type: Number
+        - contextPath: OctoxLabs.Discovery.hr_status
+          description: 'Last discovery human readable status'
+          type: String
+        - contextPath: OctoxLabs.Discovery.progress
+          description: 'Last discovery progress'
+          type: Number
+    - name: search-devices
+      deprecated: false
+      description: 'Search in your devices'
+      execution: false
+      arguments:
+        - default: null
+          description: 'Query'
+          isArray: false
+          name: query
+          required: false
+          secret: false
+        - default: null
+          description: 'Fields'
+          isArray: true
+          name: fields
+          required: false
+          secret: false
+        - default: null
+          description: 'Page'
+          isArray: false
+          name: page
+          required: false
+          secret: false
+        - default: null
+          description: 'Size'
+          isArray: false
+          name: size
+          required: false
+          secret: false
+        - default: null
+          description: 'Specific Discovery Id'
+          isArray: false
+          name: discovery_id
+          required: false
+          secret: false
+      outputs:
+        - contextPath: OctoxLabs.Devices.count
+          description: 'Total devices count'
+          type: Number
+        - contextPath: OctoxLabs.Devices.results
+          description: 'List<Dict> Device information'
+          type: Unknown
+    - name: get-device
+      deprecated: false
+      description: 'Fetch your device'
+      execution: false
+      arguments:
+        - default: false
+          required: true
+          isArray: false
+          name: hostname
+          description: 'Your device hostname'
+          secret: false
+        - default: false
+          required: false
+          isArray: false
+          name: discovery_id
+          description: 'Your device at specific discovery'
+          secret: false
+      outputs:
+        - contextPath: OctoxLabs.Device
+          description: '<Dict> Octoxlabs Device'
+          type: Unknown
+  isfetch: false
+  runonce: false
+  script: '-'
+  type: python
+  subtype: python3
+  dockerimage: demisto/octoxlabs:1.0.0.33874
+fromversion: 5.5.0
```

Comment:
Please make sure the version is the  same as you're working on (this should be the minimal versions)